### PR TITLE
Dev 28208 include clinic name

### PR DESCRIPTION
**Pull request by grossb1**
## Description of change

All appointment v2 endpoint responses have been supplemented with the clinic name (as `stationName`).  In the event that the request for the clinic name fails, the appointment will be returned without the `stationName` field.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Things to know about this PR

<!-- Please describe testing done to verify the changes or any testing planned. 
Tests were added/updated to reflect this change

### CODE REVIEW

File: modules/vaos/app/services/vaos/v2/appointments_service.rb [hunk 1]
```diff
@@ -39,7 +39,7 @@ def post_appointment(request_object_body)
         end
       end
 
-      def update_appointment(appt_id:, status:)
+      def update_appointment(appt_id, status)
```
Comment (ajmagdub) [inline modules/vaos/app/services/vaos/v2/appointments_service.rb:42]: curious why you removed the keyword parameters

File: modules/vaos/app/controllers/vaos/v2/appointments_controller.rb [hunk 2]
```diff
@@ -46,21 +64,47 @@ def appointment
...
+            appt[:station_name] = cached_clinic_names[appt[:clinic]] if cached_clinic_names[appt[:clinic]]
+          end
+        end
       end
 
-      def cancellation_reason
-        params.require(:reason)
+      def get_clinic_name(location_id, clinic_id)
+        clinics = systems_service.get_facility_clinics(location_id: location_id, clinic_ids: clinic_id)
+        clinics.first[:station_name]
```
Comment (ajmagdub) [inline modules/vaos/app/controllers/vaos/v2/appointments_controller.rb:93]: so the assumption is that the only clinic that you're interested in will be the first one returned?

**Review by ajmagdub — APPROVED** [review]
approved with a couple of comments

File: modules/vaos/app/services/vaos/v2/appointments_service.rb (hunk 1 above)
Comment (grossb1) [reply modules/vaos/app/services/vaos/v2/appointments_service.rb:42]: For consistency atm.  I like the explicit field naming better, I'll get a ticket in to change it.

File: modules/vaos/app/controllers/vaos/v2/appointments_controller.rb (hunk 2 above)
Comment (grossb1) [reply modules/vaos/app/controllers/vaos/v2/appointments_controller.rb:93]: That is correct, with this particular solution we will request a single clinic at a time so only a single clinic should be returned in the arry.

**Pull request merged**